provisioning/device/devdoc/x509_registration_requirements.md

```diff
@@ -0,0 +1,28 @@
+# X509registration requirements
+
+## Overview
+
+The `X509Registration` class is used to control the registration flow of an X509-enabled device, regardless of the transport, or `X509SecurityClient` implementation.
+
+## Example usage
+
+```js
+// TODO once all the pieces are there in the SDK
+
+```
+
+## Public API
+
+### register()
+
+The `register` method completes the authentication and registration flow for the user.
+
+**SRS_NODE_DPS_X509_REGISTRATION_18_001: [** `register` shall acquite the X509 certificate from the `X509SecurityClient` object. **]**
```

Comment:
>acquite [](start = 62, length = 7)

typo: acquiRe

Reply:
using which method? i think it's a good practice to explicitely tell what API we're supposed to use (like the next requirement does)

---
In reply to: [154875649](https://github.com/Azure/azure-iot-sdk-node/pull/169#discussion_r154875649) [](ancestors = 154875649)